### PR TITLE
Enable custom styles

### PR DESCRIPTION
**Pull request by TaiWilkin**
## Overview

Users should be able to select a color and a font in their embedded map configuration and have those settings applied throughout the application, as shown [here](https://app.abstract.com/projects/441633f0-7d8c-11e9-af1e-972cbddca60a/branches/272eff0b-3229-482f-a003-fad1e9558a87/commits/828df3ea1f6abe0ce18459a2dc1e5628e08e9df1/files/fc1791e3-b8d2-4182-b240-fef32fabe978/layers/88873DB4-D9AA-4DE4-9652-AC7ADB8C4147?collectionId=8ac19367-6932-43c2-95fa-7483af019a26&collectionLayerId=cfdc1a4d-1f4f-4fba-bcb0-4e4cb07bdaeb) and [here](https://app.abstract.com/projects/441633f0-7d8c-11e9-af1e-972cbddca60a/branches/272eff0b-3229-482f-a003-fad1e9558a87/commits/828df3ea1f6abe0ce18459a2dc1e5628e08e9df1/files/fc1791e3-b8d2-4182-b240-fef32fabe978/layers/02D8C4BE-87F3-4733-98A5-DD884FCE17D7?collectionId=8ac19367-6932-43c2-95fa-7483af019a26&collectionLayerId=1b494422-0155-4526-b816-fb927d1f5cbf).

Connects #1337, #1344

## Demo
Screen Shot 2021-05-21 at 2 07 35 PM
Screen Shot 2021-05-21 at 2 06 53 PM
Screen Shot 2021-05-21 at 3 43 17 PM
Screen Shot 2021-05-21 at 2 00 31 PM

## Testing Instructions

* Navigate to http://localhost:6543/facilities and click through to a facility’s details. 
     - [x] The website should appear as usual
* Navigate to http://localhost:6543/?contributors=4&embed=1 (use a contributor with no embed config) and click through to a facility’s details
     - [x] The embedded map version of the site should appear as usual
* Login as c2@example.com
* Navigate to embed settings and select a custom font and color
* Navigate to http://localhost:6543/?contributors=2&embed=1 and click through to a facility’s details
     - [x] The website should have the selected color and font

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines

### CODE REVIEW

**Review by caseycesari — CHANGES_REQUESTED** [review]
This is working well. 

One issue is that when you change the color/font, the preview embed map is not updated to reflect those changes. If you refresh the page, the custom colors are still not shown.

Having the preview map update with every save may be something for a follow-up issue, but if possible, the custom font/colors should be shown on the preview map after a refresh.

File: src/django/api/views.py [hunk 1]
```diff
@@ -3612,11 +3612,7 @@ def update(self, request, pk=None):
         embed_config = EmbedConfig.objects.get(id=pk)
 
         if embed_config.contributor.id is not contributor.id:
-            message = 'Expected contributor {ec},'.format(
```
Comment (TaiWilkin) [inline src/django/api/views.py]: This has been corrected in a separate PR, so is no longer needed.

File: src/app/src/components/EmbeddedMapConfigWrapper.jsx [hunk 2]
```diff
@@ -16,6 +16,10 @@ import {
 
 function EmbeddedMapConfigWrapper({ user }) {
     const [loading, setLoading] = useState(true);
+    // The iframe preview isn't aware when the embed config styles are updated.
+    // Using a timestamp in the iframe src forces a refresh when the config
+    // is changed.
+    const [timestamp, setTimestamp] = useState(Date.now());
```
Comment (caseycesari) [inline src/app/src/components/EmbeddedMapConfigWrapper.jsx:22]: Clever low-effort fix!

**Review by caseycesari — APPROVED** [review]
Good to go!

**Comment by TaiWilkin** [comment]
Thank you for the review!

**Pull request merged**